components/General/index.tsx

```diff
@@ -4,6 +4,46 @@ import Tooltip from 'antd/lib/tooltip';
 import React from 'react';
 import styled from 'styled-components';
 
+export const DateAndTime = styled(({ className, date, time }) => {
+    return (
+        <div className={className}>
+            {date}
+            <div className="secondary">{time}</div>
+        </div>
+    );
+})`
+    font-size: 1rem;
+    padding: 0.5rem 1rem;
+
+    ${(props) => {
```

Comment:
I think it was just over complicated a little. What I was more asking for was something which parses the date and time to display in that format, but didnt realise you were just using strings. But its all good I can add that.

Reply:
I also removed the parsing of color in props since I thought it was a bit pointless since if you want a different color you can just go styledDateAndTime = styled(DateAndTime)`.secondary { color: whatever; } `